app/helpers/patients_helper.rb

```diff
@@ -1,2 +1,17 @@
 module PatientsHelper
+
+  def bootstrap_class_for flash_type
```

Comment:
Use parens for function definitions?

  def bootstrap_class_for(flash_type)

seems to better follow the ruby style guide... https://github.com/bbatsov/ruby-style-guide